meteor/sofie-server/business-logic/services/timeout-auto-next-timer-service.ts

```diff
@@ -0,0 +1,38 @@
+import { AutoNextTimerService } from './interfaces/auto-next-timer-service'
+
+export class TimeoutAutoNextTimerService implements AutoNextTimerService {
+	private static instance: AutoNextTimerService
+
+	public static getInstance(): AutoNextTimerService {
+		if (!this.instance) {
+			this.instance = new TimeoutAutoNextTimerService()
+		}
+		return this.instance
+	}
```

Comment:
Can be simplified to `public static instance: AutoNextTimerService = new TimeoutAutoNextTimerService()`

Typescript/Javascript does the check to only create one instance for you already.
This was something that Anders originally showed me, and i found that i works in both Typescript/Javascript  and C#.

Reply:
That has slighly different behaviour and it's more implicit what happens. Oneliners aren't always preferable.